src/System.Security.Principal.Windows/src/System/Security/Principal/WindowsIdentity.cs

```diff
@@ -236,6 +236,7 @@ private void CreateFromToken(IntPtr userToken)
                 throw new SecurityException(new Win32Exception().Message);
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2229", Justification = "Public API has already shipped.")]
```

Comment:
Nit: It'd be nice to be consistent on the wording of these justifications... these two "public API" ones are written differently.